### PR TITLE
Delete direct-message from UI

### LINKED ISSUE #120: Delete direct message app from base templates

### PR DESCRIPTION
**Pull request by yulisuliman**
- Delete direct-message url
- Delete direct-message from templates/base

fix #120

### CODE REVIEW

**Comment by MatanP12** [comment]
what about the `direct_message` application folder?

**Comment by TomerNewmanPrograms** [comment]
> what about the `direct_message` application folder?

I dont think we should delete the intire app, just the url and the base html.
We worked hard to merge it and the modeles and views should be there when we will want to add the app in the future.

**Review by MatanP12 — DISMISSED** [review]
LGTM

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]
LGTM, just link it to an issue

**Comment by yulisuliman** [comment]
> LGTM, just link it to an issue

Oops I added the wrong link, change it now :)

**Review by OmerCS8 — APPROVED** [review]
LGTM😭

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]
I think you should also update the PR's name

**Comment by yulisuliman** [comment]
> I think you should also update the PR's name

Done

**Review by TomerNewmanPrograms — APPROVED** [review]

File: direct_message/urls.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from django.urls import path
-from . import views
+# from django.urls import path
+# from . import views
```
Comment (MatanP12) [inline direct_message/urls.py:2]: I think you should remove all the comments in the code

File: direct_message/urls.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-urlpatterns = [
-    path('', views.home, name='direct-message-home'),
-]
+# urlpatterns = [
+#   path('', views.home, name='direct-message-home'),
+# ]
```
Comment (MatanP12) [inline direct_message/urls.py:6]: same

**Review by MatanP12 — CHANGES_REQUESTED** [review]

**Review by MatanP12 — DISMISSED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

File: users/test_users_views.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+
+
+@pytest.fixture
+def new_user():
```
Comment (TomerNewmanPrograms) [inline users/test_users_views.py:13]: shouldn't it be in conftest.py?

Comment (TomerNewmanPrograms) [reply users/test_users_views.py:13]: and we also moved `test_users_views `out of `users`

**Review by MatanP12 — APPROVED** [review]
LGTM!

**Review by TomerNewmanPrograms — APPROVED** [review]

File: users/test_users_views.py (hunk 3 above)
Comment (natifridman) [reply users/test_users_views.py]: Good catch!

**Review by natifridman — APPROVED** [review]

**Pull request merged**